### DISCUSSION TITLE
quel pc choisir ?

### DISCUSSION THREAD

**Discussion by ellywklt** [community/community#143003] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Bonjour à tous,

Je suis étudiant en L1 d'informatique et je souhaite acheter un pc pour mes études et mes projets perso. J'aimerais acheter le MacBook Air M1 (2020), car j'ai une bonne prise en main des outils Apple. J'ai l'intention de développer des applications mobiles (pour Android et iOS) mais je souhaite également être en mesure de suivre tranquillement mes cours à la fac. L'idée est de garder celui-ci pendant au moins 5 ans. D'où ma question, le MacBook est-il un bon investissement dans la durée ou est-ce je dois me tourner vers complètement autre chose ?

Merci d'avance à ceux d'entre vous qui prendront le temps de me répondre !

**Comment by ghostinhershell** [comment]
Hi @ellywklt, thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.

With that being said, if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as the [Apple Support Community](https://support.apple.com/fr-fr). It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
